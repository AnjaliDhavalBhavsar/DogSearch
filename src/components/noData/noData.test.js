import renderer from "react-test-renderer";
import NoDataAvailable from ".";

test("NoDataAvailablesnapshot testing", () => {
  const element = renderer.create(<NoDataAvailable />).toJSON();
  expect(element).toMatchSnapshot();
});
