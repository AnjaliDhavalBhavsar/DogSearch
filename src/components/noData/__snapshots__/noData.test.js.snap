// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoDataAvailablesnapshot testing 1`] = `
<div
  className="containerNoData"
>
  <img
    alt="No Data"
    height="400"
    src="noData.png"
    width="400"
  />
  <div>
    No Data Available
  </div>
</div>
`;
